pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
```

Comment:
helm is built as a static binary:
```
$ ldd helm
	not a dynamic executable

```

Feel free to assign the issue to me. Will be good for my nix-education :).